src/plugins/flexsearch.js

```diff
@@ -56,6 +56,9 @@ async function createFlexsearchIndexData() {
     const vfile = await processor.process(content);
     // Add all other headings and their descriptions to the index
     vfile.data.sections.forEach(({ title, description }) => {
+      if([ 'Attributes', 'Errors' ].includes(title)) {
```

Comment:
Could we just omit all H3 headings?

Reply:
I'm not sure about omitting all - an example might be in payment requests we have an H3 which is 'Payment Activity Types'. This would be good to have in our search